src/main/kotlin/com/zepben/evolve/database/sqlite/readers/NetworkCIMReader.kt

```diff
@@ -305,6 +304,23 @@ class NetworkCIMReader(private val networkService: NetworkService) : BaseCIMRead
         return TownDetail(townName ?: "", stateOrProvince ?: "")
     }
 
+    private fun loadStreetDetail(table: TableStreetAddresses, resultSet: ResultSet): StreetDetail? {
+        val buildingName = resultSet.getNullableString(table.BUILDING_NAME.queryIndex)
+        val floorIdentification = resultSet.getNullableString(table.FLOOR_IDENTIFICATION.queryIndex)
+        val name = resultSet.getNullableString(table.NAME.queryIndex)
+        val number = resultSet.getNullableString(table.NUMBER.queryIndex)
+        val suiteNumber = resultSet.getNullableString(table.SUITE_NUMBER.queryIndex)
+        val type = resultSet.getNullableString(table.TYPE.queryIndex)
+        val displayAddress = resultSet.getNullableString(table.DISPLAY_ADDRESS.queryIndex)
+
+        if ((buildingName == null) && (floorIdentification == null) && (name == null) && (number == null) && (suiteNumber == null) && (type == null) &&
+            (displayAddress == null))
+            return null
+
+        return StreetDetail(buildingName ?: "", floorIdentification ?: "", name ?: "", number ?: "",
```

Comment:
If wrapping a line please put all args on seperate lines in the JVMM SDK.

In this case if you use the two suggestions above you can reduce this line to 

```
return StreetDetail(buildingName, floorIdentification, name, number, suiteNumber, type, displayAddress).takeUnless { it.nullOrEmpyCheckFunctionNameYouChoseThatIsBetterThanThis() }
```

You could also inline the reading of values from the `resultSet` and convert the function to an expresion body